opensrp-app/res/values/strings.xml

```diff
@@ -2,20 +2,18 @@
 <resources>
     <string name="empty_string"></string>
     <string name="app_name">OpenSRP</string>
-    <string name="userNameEditText">Enter user name</string>
-    <string name="passwordEditText">Enter password</string>
+    <string name="userNameEditText">Enter Username</string>
+    <string name="passwordEditText">Enter Password</string>
     <string name="login_text">Log In</string>
     <string name="logout_text">Logout</string>
-    <string name="image_description">logo</string>
-    <string name="spacer">spacer</string>
+    <string name="image_description">Logo</string>
+    <string name="spacer">Spacer</string>
     <string name="loggin_in_dialog_title">Logging in…</string>
-    <string name="loggin_in_dialog_message">Please wait</string>
-    <string name="login_failed_dialog_title">Log in failed</string>
-    <string name="login_failed_dialog_message">Please check the credentials</string>
-    <string name="form_back_confirm_dialog_message">Are you sure you want to go back? All the data you have entered in
-        this form will be cleared.
-    </string>
-    <string name="form_back_confirm_dialog_title">Confirm form close</string>
+    <string name="loggin_in_dialog_message">Please Wait</string>
+    <string name="login_failed_dialog_title">Log In Failed</string>
+    <string name="login_failed_dialog_message">Please Check The Credentials</string>
```

Comment:
@craigappl please allow me to ask, shouldn't this be sentence case? This and the "Log in failed", and "Please wait" Other sentences conform to the sentence case....

Reply:
It's a title, so we recommend capitalizing everything in the titles and use sentence case in the body of the message.